maud/tests/errors.rs

```diff
@@ -0,0 +1,9 @@
+use trybuild::TestCases;
+
+use std::path::PathBuf;
```

Comment:
```
warning: unused import: `std::path::PathBuf`
 --> maud/tests/errors.rs:3:5
  |
3 | use std::path::PathBuf;
  |     ^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
```

---

I'm surprised that the `cargo clippy -- -D warnings` in `.travis.yml` didn't catch this. Can you please check if there's an easy way to lint tests as well? (If it's too hard then no worries – I can split out a new issue instead.)

Reply:
```
cargo clippy --tests -- -D warnings
``` 

Be aware that `--tests` checks only the tests apparently, not the lib itself, so you're going to need two invocations:
```
cargo clippy -- -D warnings
cargo clippy --tests -- -D warnings
```

I had a bit of headache trying to figure it out four months ago. Maybe it has changed, don't know.

Reply:
Makes sense, thanks! I've filed this as #209.